quesma/model/expr_string_renderer.go

```diff
@@ -202,7 +202,10 @@ func (v *renderer) VisitSelectCommand(c SelectCommand) interface{} {
 			}
 		}
 		if len(innerColumn) == 0 {
-			innerColumn = append(innerColumn, "1")
```

Comment:
We should improve innerColumns logic. Probably also move to separate function:
- go through whole tree (not just nodes) and find `ColumnRef`
- also check Group BY and WHERE